.github/workflows/llvm-project-epoch-one.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           arch: ${{ matrix.arch }}
           file: Dockerfile.epoch1
           platforms: ${{ matrix.platforms }}
-          tags: ghcr.io/clangbuiltlinux/llvm-project:stage2
+          tags: ghcr.io/clangbuiltlinux/llvm-project:stage2-${{ matrix.arch }}
```

Comment:
Having to specify the BASE manually also isn't great for my local development workflow.

```sh
$ docker build -t stage3 . -f Dockerfile.epoch3
```
will become
```sh
$ docker build -t stage3 . -f Dockerfile.epoch3 --build-arg BASE=ghcr.io/clangbuiltlinux/llvm-project:stage2-$(uname -m)`
```
:-/

I guess I should just use the make targets.
```sh
$ make epoch3
```
So nvm; that's actually an improvement.

I guess the annoying part of Docker is that we can't execute commands in an ARG directive; `$(uname -m)` would be pretty handy...

Reply:
Reading through https://medium.com/icetek/understanding-how-docker-multi-arch-images-work-9a7e035e2868, it sounds like it should be possible to create arch-suffix-less tags...it sounds like you do create them in the first place, then use manifests to merge them? (IIUC)

https://medium.com/icetek/understanding-how-docker-multi-arch-images-work-9a7e035e2868
https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

Is the plan to implement manifests AFTER this PR?

Reply:
> Mind updating the README then; the tags will now need the arch suffix.

Sure.

> I guess I should just use the make targets.

Right, that was the entire point of wiring up the `Makefile` in this way so that we don't have to worry about the "under the hood" details.

> I guess the annoying part of Docker is that we can't execute commands in an ARG directive; `$(uname -m)` would be pretty handy...

Right, I tried that initially and was sorely disappointed when I found out it wasn't possible because it would have simplified that a lot :(

> Is the plan to implement manifests AFTER this PR?

Aha, I did not realize that creating the manifest would remove/hide the architecture specific tags. I can definitely look into implementing manifests after this, although I guess it depends on whether or not we intend for these images to be consumed by something other than this build process. I had assumed that these builds would ultimately be consumed via tarballs, rather than the Docker image, but maybe I am not thinking of a use case that you are?

Reply:
> although I guess it depends on whether or not we intend for these images to be consumed by something other than this build process

Yeah that's fair.  Ok not high priority to do so since these should only be seen within the docker files. Maybe for the final stage, having a more concise arch-free name might be nice. For the individual epochs...eh.

Ok, LGTM; thanks for getting this all wired up! This should unblock me for stage4 so let's get it merged up!

Reply:
> Maybe for the final stage, having a more concise arch-free name might be nice. For the individual epochs...eh.

Ack, let's revisit manifests then!

Merging now then I'll kick off all the builds.